src/AaveV2/mappings/Token.ts

```diff
@@ -83,13 +89,44 @@ function handleAaveV2Token(
     }
   }
 
+  //get aEmissionsPerSecond
+  let tried_getIncentivesController = tokenContract.try_getIncentivesController();
+  if (tried_getIncentivesController.reverted) log.error("tokenContract at {} call getIncentivesController() reverted", [ tokenContract._address.toHex() ]); 
```

Comment:
I would prefer `{` blocks everywhere.

Reply:
sure, me as well - I was copying the convention that was used in the previous code - I will add the block there too